Documentation/usage/dlv_dap.md

```diff
@@ -17,10 +17,20 @@ The server does not yet accept multiple client connections (--accept-multiclient
 While --continue is not supported, stopOnEntry launch/attach attribute can be used to control if
 execution is resumed at the start of the debug session.
 
+The --connect flag is a special flag that makes the server operate in reverse mode.
+In this mode, Delve connects to a DAP client listening on host:port,
+instead of listening for connections.
+
 ```
 dlv dap
 ```
 
+### Options
+
+```
+      --connect string   host:port of the DAP client when running in reverse mode.
```

Comment:
It's reverse mode because the connection set up is initiated in the opposite way and this is not the conventional client-server flow. It's not tied to the RunInTerminal request concept. Anyway, doc was re-written.